package.json

```diff
@@ -208,5 +209,10 @@
     "tty": "tty-browserify",
     "vm": "vm-browserify",
     "tls": false
+  },
+  "husky": {
```

Comment:
I think it would be nicer to have linter auto-fixing the issues and adding them to _git stage_ then committing.
Do you think it's worthwhile to have a look at the following => https://github.com/okonet/lint-staged